### PR TITLE
Cross ref aug22

### PR DESCRIPTION
**Pull request by zspitzer**

### CODE REVIEW

File: docs/00.home/homepage.md [hunk 1]
```diff
@@ -8,9 +8,24 @@ Welcome to the Official [Lucee Server](https://lucee.org) Documentation.
 
 ---
 
+Lucee is a [high performance](https://community.ortussolutions.com/t/how-does-cfml-really-perform-compared-to-other-languages/9325), [open source](https://github.com/lucee/Lucee), ColdFusion / cfml server engine, written in Java.
```
Comment (jbampton) [inline docs/00.home/homepage.md:11]:
```suggestion
Lucee is a [high performance](https://community.ortussolutions.com/t/how-does-cfml-really-perform-compared-to-other-languages/9325), [open source](https://github.com/lucee/Lucee), ColdFusion / CFML server engine, written in Java.
```

File: docs/03.reference/01.functions/arrayfindall/_arguments/value_or_closure.md [hunk 2]
```diff
@@ -1 +1,4 @@
-The value to find or a closure/function that gets every value of the array as input and returns true if the given value is right.
\ No newline at end of file
+- The value to find 
+- A closure/function that gets every value of the array as input and returns true if the given value is right, with with the following signature:
```
Comment (jbampton) [inline docs/03.reference/01.functions/arrayfindall/_arguments/value_or_closure.md:2]:
```suggestion
- A closure/function that gets every value of the array as input and returns true if the given value is right, with the following signature:
```

File: docs/03.reference/05.objects/array/findall/_arguments/value_or_closure.md [hunk 3]
```diff
@@ -1 +1,4 @@
-The value to find or a closure/function that gets every value of the array as input and returns true if the given value is right.
\ No newline at end of file
+- The value to find 
+- A closure/function that gets every value of the array as input and returns true if the given value is right, with with the following signature:
```
Comment (jbampton) [inline docs/03.reference/05.objects/array/findall/_arguments/value_or_closure.md:2]:
```suggestion
- A closure/function that gets every value of the array as input and returns true if the given value is right, with the following signature:
```

**Comment by zspitzer** [comment]
thanks @jbampton only on my first coffee today

**Pull request merged**